apps/block_scout_web/lib/block_scout_web/controllers/api/rpc/reward_controller.ex

```diff
@@ -126,4 +172,55 @@ defmodule BlockScoutWeb.API.RPC.RewardController do
   defp to_address_hash(address_hash_string, key) when key == "validatorAddress" do
     {:validator_format, Chain.string_to_address_hash(address_hash_string)}
   end
+
+  defp to_address_hashes(address_hashes_string, key) when key == "voterAddresses" do
```

Comment:
These three methods seem to be copy + pasted, with the only difference being the atom that's returned for the success / fail tag. Let's implement them with shared functionality to reduce duplication